### DISCUSSION TITLE
Welcome to FlorisBoard Discussions!

### DISCUSSION THREAD

**Discussion by patrickgold** [florisboard/florisboard#1185] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Engage with other community members.
  * Give general feedback about this project
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  If you want to share an idea for a new feature or want to report a bug/crash, please continue to file an issue per bug/feature!

  Also consider joining the FlorisBoard Matrix community channel:  
[![Matrix badge](https://img.shields.io/badge/chat-%23florisboard%3amatrix.org-blue)](https://matrix.to/#/#florisboard:matrix.org)

**Comment by ruyuma44-tech** [comment]
My keyboard suggestion is not working
I frequently use it 
If anyone has a solution ?

**Reply by lm41** [reply]
Suggestions are not implemented.
